src/SlnParser/Contracts/BuildConfigurationDefaults.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace SlnParser.Contracts
+{
+    /// <summary>
+    ///     Common build-configuration defaults
+    /// </summary>
+    public static class BuildConfigurationDefaults
+    {
+        /// <summary>
+        ///     The debug configuration
+        /// </summary>
+        public const string Debug = "Debug";
+
+        /// <summary>
+        ///     The release configuration
+        /// </summary>
+        public const string Release = "Release";
+
+        /// <summary>
+        ///     The default configuration
+        /// </summary>
+        public const string Default = "Default";
```

Comment:
I don't see where this is used

Reply:
Just thought this might be a helpful thing to have for users, if they went to check certain things. Let me know if this might not be needed though.